package.json

```diff
@@ -87,10 +94,11 @@
         "babel-preset-env": "^1.7.0",
         "bufferutil": "^4.0.1",
         "cross-env": "^5.1.1",
-        "electron": "11.1.0",
+        "electron": "^8.2.0",
```

Comment:
@TriPhan96 electron version should be 11.1.0

Reply:
Hi @bassemmagdy, we have upgrade the Electron package to [version 11.1.0](https://github.com/FDS-Ledger/symbol-desktop-wallet/blob/fb1ac37cb2424fa239eabd7d63529f30482e41e0/package.json#L101). Please have a look at this PR, thank you.